sycl/test-e2e/Matrix/get_coord_bf16_matC_impl.hpp

```diff
@@ -0,0 +1,102 @@
+//==----------- get_coord_bf16_matC_impl.hpp  - DPC++ joint_matrix---------==//
```

Comment:
This test should be called get_coord_float_matC then, right?

Reply:
Yep, good catch :)